web/src/helpers/cadPackages/openScadController.js

```diff
@@ -1,19 +1,24 @@
+// const openScadBaseURL = 'http://localhost:8080' // for local development
 const openScadBaseURL =
   'https://x2wvhihk56.execute-api.us-east-1.amazonaws.com/dev'
 
 export const render = async ({ code, settings }) => {
+  const body = JSON.stringify({
+    settings: {
+      size: {
+        x: 500,
+        y: 500,
+      },
+      camera: settings.camera,
+    },
+    file: code,
+  })
   const response = await fetch(openScadBaseURL + '/render', {
     method: 'POST',
-    headers: new Headers().append('Content-Type', 'application/json'),
```

Comment:
express did not like this `new Hearders` business, and aws seems to be fine with regular json so I've changed it 🤷